cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ZookeeperCluster.java

```diff
@@ -58,11 +56,12 @@
     public static final String KEY_STORAGE = "zookeeper-storage";
     public static final String KEY_JVM_OPTIONS = "zookeeper-jvmOptions";
     public static final String KEY_RESOURCES = "zookeeper-resources";
+    public static final String KEY_ZOOKEEPER_CONFIG = "zookeeper-config";
 
-    // Zookeeper configuration keys
-    private static final String KEY_ZOOKEEPER_NODE_COUNT = "ZOOKEEPER_NODE_COUNT";
-    public static final String KEY_ZOOKEEPER_METRICS_ENABLED = "ZOOKEEPER_METRICS_ENABLED";
-    public static final String KEY_KAFKA_HEAP_OPTS = "KAFKA_HEAP_OPTS";
+    // Zookeeper configuration keys (EnvVariables)
+    private static final String ENV_VAR_ZOOKEEPER_NODE_COUNT = "ZOOKEEPER_NODE_COUNT";
+    public static final String ENV_VAR_ZOOKEEPER_METRICS_ENABLED = "ZOOKEEPER_METRICS_ENABLED";
+    protected static final String ENV_VAR_ZOOKEEPER_CONFIGURATION = "ZOOKEEPER_CONFIGURATION";
```

Comment:
We've got 3 different constants with 3 different visibility mods here: Can we decide on a single modifier?